src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2381,6 +2396,17 @@ public function attributesToArray()
 	{
 		$attributes = $this->getArrayableAttributes();
 
+		// If the attribute exists within the "encrypt" array, we will decrypt
+		// it here upon retrieval, so that the developers may have peace of
+		// mind knowing that the unencrypted value isn't actually stored.
+		foreach ($attributes as $key => $value)
+		{
+			if (in_array($key, $this->encrypt))
+			{
+				$attributes[$key] = $this->decrypt($value);
+			}
+		}
```

Comment:
Instead of looping over all attributes I'd rather just iterate `$this->encrypt` and check if the attribute exists. Like it is done with the dates on the next few lines...